### PR TITLE
Added fee ids to MasterPricer

### PR DESCRIPTION
**Pull request by bimusiek**
Hey, I felt bad about just asking so I wanted to contribute a little bit.

Here is option to add FeeIds when searching in MasterPricer.

Possible options I saw in Amadeus.
```
FFI - shows fare family names from airlines (ex. Economy Saver)
UPH/UPB - upsell, so each recommendation gets reference to more expensive fares
```
Tell me if you want me to change anything! 

Cheers, Mike.

### CODE REVIEW

**Comment by DerMika** [comment]
Well, this is wonderful, thanks for contributing! :)

I'll go through the code tomorrow to see if everything looks good.

File: src/Amadeus/Client/Struct/Fare/MasterPricer/FareOptions.php [hunk 1]
```diff
@@ -79,6 +80,24 @@ public function __construct(array $flightOptions, array $corpCodesUniFares, $tic
...
         $this->loadCurrencyOverride($currency);
+        if (!is_null($feeIds)) {
+            $this->loadFeeIds($feeIds);
+        }
+    }
+
+    /**
+     * Set fee ids if needed
+     *
+     * @param string|null $currency
```
Comment (DerMika) [inline src/Amadeus/Client/Struct/Fare/MasterPricer/FareOptions.php]: Looks like this PHPDoc needs some love.

File: src/Amadeus/Client/RequestOptions/Fare/MPFeeId.php [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+/**
+ * MasterPricer Fee Id request options.
+ *
+ * @package Amadeus\Client\RequestOptions\Fare
+ * @author Dieter Devlieghere <dieter.devlieghere@benelux.amadeus.com>
+ */
+class MPFeeId extends LoadParamsFromArray
+{
+    /**
+     * Fee Type
```
Comment (DerMika) [inline src/Amadeus/Client/RequestOptions/Fare/MPFeeId.php]: I'd like to have all possible Fee Types as constants "TYPE_*" here so users know what values are possible (and supported by Amadeus). 

This is analogous what I do in other places, for example [here](https://github.com/amabnl/amadeus-ws-client/blob/master/src/Amadeus/Client/RequestOptions/Fare/MasterPricer/FFCriteria.php#L35)

Here's the list of possible fee types: 

```
Fare_MasterPricerTravelBoardSearch/fareOptions/feeIdDescription/feeId/feeType
Attribute type identification, coded codesets
Value	Description
ARP	Arp number
FBA	Free Baggage Allowance
FFI	Fare Family Information
NPS	No PNR Split
PTRAM	Price To Reach amount type
RTC	Half round trip combination
SBF	Search by FBA
SORT	Sorting option (FEE, NOFEE)
TOKEN	Token
UPB	Upsell per bound
UPH	Homogenous upsell
```

As seen here: https://webservices.amadeus.com/extranet/structures/viewMessageStructure.do?id=5769&serviceVersionId=4507&isQuery=true

File: src/Amadeus/Client/Struct/Fare/MasterPricer/FareOptions.php [hunk 3]
```diff
@@ -62,8 +62,9 @@ class FareOptions
      * @param array $corpCodesUniFares list of Corporate codes for Corporate Unifares
      * @param bool $tickPreCheck Do Ticketability pre-check?
      * @param string|null $currency Override Currency conversion
+     * @param array|null $flightOptions List of FeeIds
```
Comment (DerMika) [inline src/Amadeus/Client/Struct/Fare/MasterPricer/FareOptions.php]: I'd prefer to have the typed objects here, e.g. `FeeId[]`

File: src/Amadeus/Client/Struct/Fare/MasterPricer/FeeId.php [hunk 4]
```diff
@@ -33,4 +33,20 @@ class FeeId
     public $feeType;
```
Comment (DerMika) [inline src/Amadeus/Client/Struct/Fare/MasterPricer/FeeId.php]: `$feeType` and `$feeIdNumber` need phpdocs.

Also here I'd like to have the same list of constants of possible feeTypes.

**Comment by bimusiek** [comment]
@DerMika Hey, I believe I did all changes you requested :) Tell me please if I missed something.

**Comment by bimusiek** [comment]
@DerMika Updated.

**Comment by DerMika** [comment]
Wonderful, Thanks again for your help! I've merged your PR.

**Pull request merged**